Examples/SAM/Tests/LambdaTests/data/apiv2.json

```diff
@@ -0,0 +1 @@
+{"version":"2.0","queryStringParameters":{"arg2":"value2","arg1":"value1"},"isBase64Encoded":false,"requestContext":{"timeEpoch":1671601639995,"apiId":"x6v980zzkh","http":{"protocol":"HTTP\/1.1","userAgent":"Mozilla\/5.0 (Macintosh; Intel Mac OS X 10.15; rv:108.0) Gecko\/20100101 Firefox\/108.0","sourceIp":"90.103.90.59","path":"\/test","method":"GET"},"domainName":"x6v980zzkh.execute-api.eu-central-1.amazonaws.com","accountId":"486652066693","requestId":"de2cRil5liAEM5Q=","time":"21\/Dec\/2022:05:47:19 +0000","domainPrefix":"x6v980zzkh","stage":"$default"},"rawPath":"\/test","headers":{"accept-encoding":"gzip, deflate, br","host":"x6v980zzkh.execute-api.eu-central-1.amazonaws.com","accept-language":"en-US,en;q=0.8,fr-FR;q=0.5,fr;q=0.3","sec-fetch-dest":"document","x-amzn-trace-id":"Root=1-63a29de7-371407804cbdf89323be4902","x-forwarded-for":"90.103.90.59","accept":"text\/html,application\/xhtml+xml,application\/xml;q=0.9,image\/avif,image\/webp,*\/*;q=0.8","sec-fetch-site":"none","content-length":"0","user-agent":"Mozilla\/5.0 (Macintosh; Intel Mac OS X 10.15; rv:108.0) Gecko\/20100101 Firefox\/108.0","sec-fetch-user":"?1","x-forwarded-port":"443","dnt":"1","x-forwarded-proto":"https","sec-fetch-mode":"navigate","upgrade-insecure-requests":"1"},"rawQueryString":"arg1=value1&arg2=value2","routeKey":"$default"}
```

Comment:
consider formatting this for readability

Reply:
done